include/boost/math/fft/real_algorithms.hpp

```diff
@@ -351,7 +351,7 @@
             const T *in_first, 
             const T *in_last, 
             T* out, 
-            int sign,
+            int /*sign*/,
```

Comment:
The sign for real transforms is not important. The default behavior is that the real-to-complex is a forward transform.